src/graphdb.js

```diff
@@ -150,36 +184,59 @@ function graphDBInferenceSchema (graphbSchema, addMutations) {
         r += `input ${nodeCase}Input {\n`;
         r += '\t_id: ID @id\n';
         node.properties.forEach(property => {
-            r+= `\t${property.name}: ${property.type}\n`;
+            let propertyCase = replaceCleanseLabel(property.name);
+            if (property.name !== propertyCase) {
+                r+= `\t${propertyCase}: ${property.type} @alias(property: "${property.name}")\n`;
+            }
+            else {
+                r+= `\t${property.name}: ${property.type}\n`;
+            }
         });
         r += '}\n\n';
     })
 
     // edge types
     gdbs.edgeStructures.forEach(edge => {
         // edge type
-        let edgeCase = edge.label;
+        let edgeCase = replaceCleanseLabel(edge.label);
         if (changeCase) {
-            edgeCase = toPascalCase(edge.label);
-            r += `type ${edgeCase} @alias(property:"${edge.label}") {\n`;        
-        } else {
+            edgeCase = toPascalCase(edgeCase);
+        }
+        if (edge.label !== edgeCase) {
+            r += `type ${edgeCase} @alias(property:"${edge.label}") {\n`;  
+        }
+        else {
             r += `type ${edgeCase} {\n`;
         }
         r += '\t_id: ID! @id\n';
 
         edge.properties.forEach(property => {
-            if (property.name == 'id')                
-                r+= `\tid: ID\n`;
-            else
-                r+= `\t${property.name}: ${property.type}\n`;
+            if (property.name == 'id') {
```

Comment:
This is likely **not** an issue, but you may want to use triple equals for safety here.

Rarely, if ever, should you use double equals. I basically only use it for checking for null or undefined.